pipeline/transformer/transformer_calm/src/main/scala/uk/ac/wellcome/platform/transformer/calm/CalmTransformer.scala

```diff
@@ -33,6 +33,15 @@ object CalmTransformer
     ("Arrangement", ArrangementNote(_))
   )
 
+  // As much as it might not look like it, these values mean it should
+  // not be suppressed.
+  val nonSupressedStatus = List(
```

Comment:
I might prefer to do this as a pattern match with a separate branch for each string (rather than `.contains` or `case "a" | "b" | "c" =>`, but if it's to stay like this
```suggestion
  val nonSuppressedStatuses = List(
```

Reply:
Ha deja vu 
https://github.com/wellcomecollection/catalogue/pull/619#discussion_r435095448

Reply:
Ha, at least I'm picking the same nits